### PR TITLE
Edge/cie 3986 migration procedure update

### PR DESCRIPTION
**Pull request by sharath-jk**

### CODE REVIEW

File: content/edge/migration-bundle/restoring-the-data-on-edge-109.md [hunk 1]
```diff
@@ -62,18 +62,24 @@ mongofiles -d management --prefix cmdata get  112 -l /tmp/apps/cockpit.zip
...
+```shell
+rpm -ivh zip-3.0-11.el7.x86_64.rpm
+```
+
 5. Prepare the applications for deployment using the commands:
 
-    {{< c8y-admon-important >}}
-Do not include the *streaming-analytics-app.zip* file in the ZIP package.
-    {{< /c8y-admon-important >}}
+    {{< c8y-admon-important >}}Do not include the *streaming-analytics-app.zip* file in the ZIP package. {{< /c8y-admon-important >}}
```
Comment (chandrababumerla) [inline content/edge/migration-bundle/restoring-the-data-on-edge-109.md:82]: Here we are referring "package-cumulocity-$UI_VERSION.zip" as "ZIP package", which is fine I think. But in step #4 above we are referring "zip-3.0-11.el7.x86_64.rpm" as "ZIP package", which instead probably we can refer to as a "zip rpm package" or "rpm package"

File: content/edge/migration-bundle/sample-scripts-to-automate-the-migration.md [hunk 2]
```diff
@@ -67,6 +67,9 @@ Before copying the backup, ensure that there is sufficient disk space in your Ed
 	```shell
 	./restore.sh migration_data.tgz /home/admin/migration_data
 	```
+
+The `restore.sh` script fails if your Edge appliance is not connected to the internet. In such cases, ensure that the ZIP package is available locally, and replace the command `rpm -ivh http://mirror.centos.org/centos/7/os/x86_64/Packages/zip-3.0-11.el7.x86_64.rpm` with `rpm -ivh zip-3.0-11.el7.x86_64.rpm` in the `restore.sh` script.
```
Comment (chandrababumerla) [inline content/edge/migration-bundle/sample-scripts-to-automate-the-migration.md:71]: Same comment as above

**Review by chandrababumerla — APPROVED** [review]

File: content/edge/migration-bundle/restoring-the-data-on-edge-109.md (hunk 1 above)
Comment (sharath-jk) [reply content/edge/migration-bundle/restoring-the-data-on-edge-109.md:82]: Agree. I'll call it as RPM package.

File: content/edge/migration-bundle/sample-scripts-to-automate-the-migration.md (hunk 2 above)
Comment (sharath-jk) [reply content/edge/migration-bundle/sample-scripts-to-automate-the-migration.md:71]: Agree. I'll call it as RPM package.

**Review by bbyreddy — APPROVED** [review]

File: content/edge/migration-bundle/restoring-the-data-on-edge-109.md [hunk 3]
```diff
@@ -17,123 +17,145 @@ Perform these steps as **root** user in your Edge 10.9 appliance.
...
+
+    ```shell
+    systemctl restart cumulocity-agent
+    systemctl restart nginx
+    systemctl restart cumulocity-core-karaf
+    monit restart opcua_device_gateway_proc
+    monit restart opcua_mgmt_service_proc
+    ```
+
 12. Restore the Streaming Analytics application.
```
Comment (rssag) [inline content/edge/migration-bundle/restoring-the-data-on-edge-109.md:148]: this is quite a long list - can it be broken down with subsections maybe?

**Review by rssag — APPROVED** [review]
i changed the indentation in the long list directly via commit.

approval, but there is a comment concerning the long list.

File: content/edge/migration-bundle/restoring-the-data-on-edge-109.md (hunk 3 above)
Comment (sharath-jk) [reply content/edge/migration-bundle/restoring-the-data-on-edge-109.md:148]: It is a migration procedure and needs to be performed in exactly the same order. So, we left it as a long list.

**Pull request merged**